### PR TITLE
Editable station

### PR DESCRIPTION
**Pull request by BuckarooBanzay**
See #23 
by @SwissalpS (i took the liberty of creating a PR ;)
Current state available on `test.pandorabox.io`

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
Looks like the `timestamp` variable is being used, while LuaCheck reports it's not 👀 Might want to disable that for that line https://github.com/mt-mods/travelnet/runs/4544141062?check_suite_focus=true

**Comment by S-S-X** [comment]
> Looks like the `timestamp` variable is being used, while LuaCheck reports it's not 👀 Might want to disable that for that line https://github.com/mt-mods/travelnet/runs/4544141062?check_suite_focus=true

`timestamp` defined on that line is unused.

File: functions.lua [hunk 1]
```diff
@@ -275,8 +303,301 @@ travelnet.remove_box = function(_, _, oldmetadata, digger)
...
+	if minetest.is_protected(pos, player_name)
+		and not minetest.check_player_privs(player_name, { protection_bypass = true })
+	then
+		minetest.record_protection_violation(pos, player_name)
+		return
+	end
+
+	local network
+	local timestamp = os.time()
+	network = travelnet.get_network(owner_name, station_network)
```
Comment (S-S-X) [inline functions.lua:556]:
```suggestion
	local network = travelnet.get_network(owner_name, station_network)
```

**Comment by Panquesito7** [comment]
> > Looks like the `timestamp` variable is being used, while LuaCheck reports it's not 👀 Might want to disable that for that line https://github.com/mt-mods/travelnet/runs/4544141062?check_suite_focus=true
> 
> `timestamp` defined on that line is unused.

Note that there's another variable with the same name in line 355 or something like that I think. If it's in the same function, then it's gonna be a problem. :)

**Comment by S-S-X** [comment]
> > > Looks like the `timestamp` variable is being used, while LuaCheck reports it's not 👀 Might want to disable that for that line https://github.com/mt-mods/travelnet/runs/4544141062?check_suite_focus=true
> > 
> > 
> > `timestamp` defined on that line is unused.
> 
> Note that there's another variable with the same name in line 355 or something like that I think. If it's in the same function, then it's gonna be a problem. :)

It is not in same function and even if it would be in same function it would be shadowing original variable and still unused (bu luacheck would also complain about shadowing / redefinition)

**Comment by SwissalpS** [comment]
> > > > Looks like the `timestamp` variable is being used, while LuaCheck reports it's not eyes Might want to disable that for that line https://github.com/mt-mods/travelnet/runs/4544141062?check_suite_focus=true
> > > 
> > > 
> > > `timestamp` defined on that line is unused.
> > 
> > 
> > Note that there's another variable with the same name in line 355 or something like that I think. If it's in the same function, then it's gonna be a problem. :)
> 
> It is not in same function and even if it would be in same function it would be shadowing original variable and still unused (bu luacheck would also complain about shadowing / redefinition)

must be from copy paste travelnet to elevators operation. Elevators are not sorted by timestamp iirc.

**Comment by SwissalpS** [comment]
It seems that when you punch a box and then press the edit button nothing happens. Only when you right-click the box does the edit button work. Is that just me or are others having the same issue?

**Comment by BuckarooBanzay** [comment]
> It seems that when you punch a box and then press the edit button nothing happens. Only when you right-click the box does the edit button work. Is that just me or are others having the same issue?

this was happening before too under certain conditions, i don't think thats a big issue
LGTM :+1:

**Pull request merged**